### DISCUSSION TITLE
osu!mania on iPad is hardly readable with redirected stages

### DISCUSSION THREAD

**Discussion by Nana-Sakura** [ppy/osu#29587] [Q&A]
![IMG_1341](https://github.com/user-attachments/assets/d535ea7d-5659-4647-b315-a2d752a324ee)
As can be seen in this image, 4 tapping stages are there. While tapping, the 4 stages are constantly changes from partially transparent and opaque, with flashing white lights, making it actually very hard to read the map, let alone play that. 
Another issue with the stage is that the tapping stages are anti-mapping (maybe say mapping, e.g. mapping four fingers to dfjk keys), in this image the conveyor belt (maybe say rail or trail idk) is gathered centrally, while the tapping stages are forcing players to make their hands away from center, may be of counter-intuitive. 
So I will suggest remove that stage, cuz I really find difficulty with pass any map on my iPad now, or I may have to go back to osu!stable, simply play with keyboards.

**Comment by peppy** [comment]
In the near future you will be able to customise this overlay in the skin editor. Stay tuned for that.
